tests/testthat/test-luz.R

```diff
@@ -0,0 +1,123 @@
+skip_on_cran()
+skip_if_not_installed(pkg = c("torch", "luz", "plumber"))
+torch::install_torch()
+
+fitted <- torch::nn_linear %>%
+    luz::setup(loss = torch::nnf_mse_loss, optimizer = torch::optim_sgd) %>%
+    luz::set_hparams(in_features = 10, out_features = 1) %>%
+    luz::set_opt_hparams(lr = 0.01) %>%
+    luz::fit(list(torch::torch_randn(100, 10), torch::torch_randn(100, 1)), verbose = FALSE)
+
+v <- vetiver_model(
+    fitted,
+    "luz-model",
+    prototype_data = torch::torch_randn(10, 10)
+)
```

Comment:
Can you share an example of how to train a model with rectangular data like mtcars or [like this one in this README](https://brulee.tidymodels.org/#example)? That will be a better fit for testing for now.